Create ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md (#1986)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1019-22-1-disable-inventory-roles-page-for-external-users-if-inventory-5129-graft.md

---
date: ""

> title: disable inventoryRoles page for external users if inventory… (#5129) [GRAFT][release/cd] (#6157)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Disable inventory roles page for external users if inventory is disabled
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57096
version: 1019.22.1
---

> disable inventoryRoles page for external users if inventory… (#5129) [GRAFT][release/cd] (#6157)

Author: The Claude AI suggested to use the following body:
```suggestion
In an effort to improve the user experience and prevent confusion, the inventory roles page is now disabled for external users if the inventory feature is not enabled. Previously, external users were able to access the inventory roles page even if the inventory was disabled, which could lead to misunderstandings about the availability and functionality of the feature. With this change, external users will no longer see the inventory roles page when the inventory is turned off, providing a clearer and more consistent experience across the application.
```

> Administrators can no longer change the global roles and inventory roles of a user managed by the SSO server if they are updated during each login. This option can be set in the access mapping of the SSO configuration.

Reviewer: ```suggestion
Administrators can no longer change the global roles and inventory roles of users managed by the single sign-on (SSO) server if they are updated during each login. This option can be set in the access mapping of the SSO configuration.
```

> Administrators can no longer change the global roles and inventory roles of users managed by the single sign-on (SSO) server if they are updated during each login. This option can be set in the access mapping of the SSO configuration.

Reviewer: This option can be set in the access mapping of the SSO configuration > @kalavathig1 What exactly can be set there?
if they are updated during each login > What does this mean? Do the roles have to be set again or or they taken from any other source?

Reviewer 2: ```suggestion
Administrators can no longer change the global roles and inventory roles of users managed by the single sign-on (SSO) server if they are updated during each login. The option "
Roles selected in the rules below will be reassigned to a user on each log in and other ones will be cleared
" can be set in the access mapping of the SSO configuration
```

Reviewer 2: @BeateRixen please check if it makes sense?

Reviewer: Sounds much better. Please commit I I will do one more language review

> In an effort to improve the user experience and prevent confusion, the inventory roles page is now disabled for external users if the inventory feature is not enabled. Previously, external users were able to access the inventory roles page even if the inventory was disabled, which could lead to misunderstandings about the availability and functionality of the feature. With this change, external users will no longer see the inventory roles page when the inventory is turned off, providing a clearer and more consistent experience across the application.

Reviewer: ```suggestion
In an effort to improve the user experience and prevent confusion, the **Inventory roles** tab is now disabled for external users if the inventory feature is not enabled. Previously, external users were able to access the **Inventory roles** tab even if the inventory was disabled, which could lead to misunderstandings about the availability and functionality of the feature. With this change, external users will no longer see the **Inventory roles** tab when the inventory is turned off, providing a clearer and more consistent experience across the application.
```

Reviewer: What does "external user" mean here?
What does "inventory feature" mean? We should get that clearer. Is it a specific permission? What does it mean "inventory is disabled/turned off"? This is a bit fuzzy.

Reviewer: @martin-sag Please follow up on this.

Reviewer 3: @BeateRixen 
External user is the user who is managed by sso server

  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57096
version: 1019.22.1
---

> Administrators can no longer change the global roles and inventory roles of users managed by the single sign-on (SSO) server if they are updated during each login. The option "

Reviewer: ```suggestion
Administrators can no longer change the global roles and inventory roles of users managed by the single sign-on (SSO) server if they are updated during login. 
```

> Roles selected in the rules below will be reassigned to a user on each log in and other ones will be cleared" can be set in the access mapping of the SSO configuration

Reviewer: ```suggestion
The option "Roles selected in the rules below will be reassigned to a user on each log in and other ones will be cleared" can be set in the access mapping of the SSO configuration.
```
